src/ucs/type/worker_param.h

```diff
@@ -0,0 +1,19 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 20016.  ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_TYPE_CPU_PARAM_H
```

Comment:
need to not use this structure in UCT at all, so there would not be dependency issues